src/styles/about.scss

```diff
@@ -1,47 +1,89 @@
-img {
-    margin-top: 0.5rem;
-    width: auto;
-    width: 40vh;
```

Comment:
You're setting width by viewport _height_ here. It might work in some cases (aka a "magic number") but if you think about the height of a vertically-oriented iPhone screen vs. a horizontal laptop screen, you're going to get a way different result. 

vw and vh are sexy but tricky. You're better off using px or rem in most of these situations.

A better flexible solution, is to size the image's _container_ but not put a size on the image itself. Then the image will resize fluidly as the container resizes.